src/shared_modules/content_manager/tests/component/action_test.cpp

```diff
@@ -210,14 +210,18 @@ TEST_F(ActionTest, TestInstantiationOfTwoActionsWithTheSameTopicName)
     const auto& topicName {m_parameters.at("topicName").get_ref<const std::string&>()};
     const auto& outputFolder {m_parameters.at("configData").at("outputFolder").get_ref<const std::string&>()};
 
+    // Both actions can't use RocksDB.
+    auto parametersWithoutDatabasePath = m_parameters;
+    parametersWithoutDatabasePath.at("configData").erase("databasePath");
```

Comment:
What do you mean?

RocksDB is now enabled in all the test fixtures. In this specific fixture, there are two actions that try to use the same `databasePath` so I'm disabling RocksDB in one of them in order to avoid conflicts (exceptions).